agent/containers/images/visualizers/prom-graf-visualizer/combo.json

```diff
@@ -0,0 +1,14536 @@
+{"dashboard":
```

Comment:
Are these three `prom-graf-visualizer/*.json` files the same as the `live-metric-visualizer/*.json` files?

Reply:
I was thinking the same thing. They have exactly the same sizes, so I think so.

Can the be factored down a level so they're not duplicated?